node/tests/integration/main.rs

```diff
@@ -616,8 +616,8 @@ async fn assert_slot_status_fold(
     let (account, _) = client.get_account(account_id).unwrap();
     let game_account_storage = account.storage();
 
-    let fold_index = slot_data.current_turn_index() + 9;
-
+    let fold_index = slot_data.current_turn_index() + 10;

```

Comment:
Can you give this offset an appropriate name ?

Reply:
Already done in the latest PR
https://github.com/RizeLabs/aze-server/blob/c6a10c2ed0a19c3d41256e1f45b6d1c2c7338cfa/node/tests/integration/main.rs#L1070